media/editor/dataDisplay.tsx

```diff
@@ -296,7 +297,7 @@ export const DataDisplay: React.FC = () => {
 			select.messageHandler.sendEvent({
 				type: MessageType.DoCopy,
 				selections: ctx.selection.map(r => [r.start, r.end]),
-				format: ctx.focusedElement.char ? CopyFormat.Utf8 : CopyFormat.Base64,
+				format: ctx.focusedElement.char ? CopyFormat.Utf8 : copyType,
```

Comment:
The webview doesn't actually need to know about the type the user currently has selected. I would probably replace the `format` property with `text?: boolean` and then map that to whatever the setting value is in the extension side.

Reply:
i have no idea